.bash_profile

```diff
@@ -0,0 +1,48 @@
+dd `~/bin` to the `$PATH`
```

Comment:
Did you mean to check this in? On a scale from 0 to crazy this is at a solid 8 or 9